communication/calendar-guidelines.md

```diff
@@ -95,6 +100,54 @@ will prevent you from getting updates like a canceled meeting. Join the main
 
 Accept the invite from the sender and you'll have the updates.
 
+### Calendar event template
+
+In case you are creating a calendar event and wondering what to include in it, you can refer to the following template
```

Comment:
non-blocking nit, we try and word-wrap at 80 characters unless its something like a link or list object.